blockchain/indexers/common.go

```diff
@@ -143,7 +143,12 @@ type Indexer interface {
 	WaitForSync() chan bool
 
 	// Subscribers returns all client channels waiting for the next index update.
+	// Deprecated: This will be removed in the next major version bump.
 	Subscribers() map[chan bool]struct{}
+
+	// NotifySyncSubscribers signals subscribers of an index sync update.
+	// This should only be called when an index is synced.
+	NotifySyncSubscribers()
```

Comment:
This is a major change to the API that would require a major module version bump.  A major bump is needed for #2836 too, so I'll open a PR that does the bumps for that here shortly.

Reply:
This just needs to be rebased over #2903 to resolve this one.

Reply:
Noted, thanks.